graylog2-web-interface/src/views/components/aggregationbuilder/EventListConfiguration.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+
+import EventListConfiguration from './EventListConfiguration';
+
+describe('EventListConfiguration', () => {
+  afterEach(cleanup);
+
+  it('should render minimal', () => {
+    const { container } = render(<EventListConfiguration />);
+    expect(container).toMatchSnapshot();
```

Comment:
This is more a general discussion. What is the reason you are using a snapshot here? Couldn't we test the same case, by just checking if the text `Enable Event Annotation` gets rendered?

Reply:
I feel like this represents the widget better. But I am open for suggestions. Checking on the name though, I don't think it is enough.